src/assets/database.sql

```diff
@@ -15,6 +15,7 @@ CREATE TABLE `commands` (
   `tool` int(11) DEFAULT NULL,
   `type` varchar(30) DEFAULT NULL,
   `command` text DEFAULT NULL,
+  `value` text DEFAULT NULL,
```

Comment:
Not clear to me the need for the value here. Not sure why we cannot use in the way other commands were being set like with the command being something like `--count 10` (ie., a string containing the parameter name plus its value).